dashboard/src/utils/apiConstants.js

```diff
@@ -1,2 +1,4 @@
-export const LOCAL_BASE_URL = "/";
+export const LOCAL_BASE_URL = "http://10.1.170.201/";
```

Comment:
We need to get the `endpoints` change in. 😁

That will not only remove this constant, but this entire file. (In theory you could have symbols referencing the various endpoint key names, but in JSON format they're reasonably clear symbols already, like `api.register`, which we can reference, if we keep my `index.html` assignment to `window.origin` as `window.origin.api.register`...)